### PR TITLE
[OCPBUGS-8246]: OVN-K control plane re-deployment after node deletion

### PR DESCRIPTION
**Pull request by tmalove**
[OCPBUGS-8246](https://issues.redhat.com/browse/OCPBUGS-8246)

Version(s): 4.11+

**Merge review team: This PR was in merge review and a change was made to the command as a result.  Adding back to the queue.

Link to docs preview: (March 24)
http://file.rdu.redhat.com/tlove/etcd-ocpbugs-8246-tlove/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

QE review:
- [x ] QE has approved this change.

### CODE REVIEW

**Comment by tmalove** [comment]
@palonsoro Step 13 is ready for your review.

**Comment by palonsoro** [comment]
It is redacted in a bit confusing way, TBH.

It would be clearer if:
- The explanation of CNO redeploying OVN-Kubernetes control plane is not split between the paragraph before the command and the important note under the command.
- The important note under the command just warns about the execution time.

So I'd write it like this

Verify that CNO has redeployed OVN-Kubernetes control plane and it does not reference the wrong controller IP addresses anymore. To do so, check regularly the output of the command below. Wait until it returns an empty result before proceeding with the next step.
```
$ oc -n openshift-ovn-kubernetes get ds/ovnkube-master -o yaml | grep -E "${WRONG_MASTER_IP_1}|${WRONG_MASTER_IP_2}"
```

**IMPORTANT**: It can take more than 5-10 minutes for the OVN-Kubernetes control plane to be redeployed and the command above to return an empty output.

**Comment by palonsoro** [comment]
It now looks perfect to me. Thanks!

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -299,6 +299,17 @@ Perform the following step only if you are using `OVNKubernetes` network plugin.
 $ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
 ----
 
+. Verify that the Cluster Network Operator (CNO) re-deploys the OVN-Kubernetes control plane, and it no longer references the wrong controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before proceeding with the next step.
```
Comment (lahinson) [inline modules/dr-restoring-cluster-state.adoc:302]:
```suggestion
. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the wrong controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed with the next step.
```

Comment (lahinson) [reply modules/dr-restoring-cluster-state.adoc:302]: I checked the ISG "Prefixes" topic (https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes), and based on the guidance there, I think "redeployed" is correct.

**Review by lahinson — COMMENTED** [review]
Just a couple of minor comments, but LGTM overall!

**Comment by jboxman-rh** [comment]
@tmalove, do we want to define `WRONG_MASTER_IP_1` ect?

**Comment by jboxman-rh** [comment]
@geliu2016 @palonsoro - I wonder if we need to describe how to get WRONG IP? Or would the user reasonably know this already? The use of `${abc}` implies that user already has this set as a shell variable?

Thanks!

**Comment by tmalove** [comment]
> @tmalove, do we want to define `WRONG_MASTER_IP_1` ect?

Hi @jboxman-rh  I will discuss this with Pablo and update or explain if there is no change. When I get this resolved, should I add it back to the merge queue or get back with you to complete it (as it will be next week before I talk to him...different geo)? Thanks!

**Comment by jboxman-rh** [comment]
@tmalove, I'd add it back to the queue, thanks!

**Comment by palonsoro** [comment]
@jboxman-rh @tmalove In the explanation, we have basically nominated 1 control plane as the recovery host and the other 2 are the ones which aren't the recovery host. The name "wrong" comes from that:
- If we are recovering from a disaster scenario where only one control plane is available, those control planes are indeed broken, so they are "wrong".
- If those nodes are not actually broken, we are stopping them anyway.

IMHO it should be already clear what are we talking about and I already succeeded providing these steps with these explanations to other people, but I am open to rephrase this if you find a way to make this clearer.

> - I wonder if we need to describe how to get WRONG IP? Or would the user reasonably know this already?

There is no automated or "dummy command" that can get you these IPs. Basically, these IPs are the IPs of the control planes that are not the recovery control plane host, so they depend on a manual user choice (or a specific user situation). What we have to do is to make 100% clear that these IPs are the IPs of the other control plane hosts, if this is not clear enough already.

>  The use of ${abc} implies that user already has this set as a shell variable?

No, it doesn't. This is a tip I learnt from a colleague in a past job: Using bash-like placeholders, while making 100% clear what are them and that they must be replaced, allows a user to either:
- Replace them like with a `<whatever>` syntax.
- Truly export the desired values as bash variables and copy-paste the command as is.

Being able to do the latter has proved to be very comfortable to me and many people I have helped in the past. However, if this somehow goes against some documentation guideline, we can revisit it.

**Comment by palonsoro** [comment]
@jboxman-rh I guess it is best idea to just stick to the style guide of our docs instead of re-do the wheel. Maybe @tmalove can do that :slightly_smiling_face: 

For the rest, I hope it is clear.

**Comment by kelbrown20** [comment]
@tmalove Be sure to squash those commits before this can get merged

**Comment by tmalove** [comment]
@kelbrown20 The commits are squashed. Thx!

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**